app/src/main/java/com/greenfox/kalendaryo/SelectCalendarActivity.java

```diff
@@ -101,4 +106,10 @@ public void onFailure(Call<GoogleCalendarsResponse> call, Throwable t) {
             });
         }
     }
+
```

Comment:
onBackPressed again, this way we cannot go back and change the name, if we want to